manifests/0000_70_cluster-network-operator_01_egr_crd.yaml

```diff
@@ -0,0 +1,151 @@
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: EgressRouterSpec contains the configuration for an egress router.
+            properties:
+              ip:
+                description: IP configuration arguments.
+                properties:
+                  addresses:
+                    description: List of IP addresses to configure on the interface.
+                    items:
+                      description: EgressRouterAddress contains a pair of ip cidr and gateway to be configured on the router's interface
+                      properties:
+                        gateway:
+                          description: IP address of the next-hop gateway, if it cannot be automatically determined. Can be IPv4 or IPv6.
+                          type: string
+                        ip:
+                          description: IP is the address to configure on the router's interface. Can be IPv4 or IPv6.
+                          type: string
+                      required:
+                      - ip
+                      type: object
+                    type: array
+                  allowedDestinations:
+                    description: List of CIDR blocks that the pod is allowed to connect to via this interface. If not provided, the pod can connect to any destination.
+                    items:
+                      description: CIDR represents an IPv4 or IPV6 CIDR block.
+                      type: string
+                    type: array
+                required:
+                - addresses
+                type: object
+              networkInterface:
+                description: Specification of interface to create/use. e.g; macvlan
+                properties:
+                  macVLAN:
+                    description: Arguments specific to the interfaceType macvlan
+                    properties:
+                      master:
+                        description: Name of the master inteface. Need not be specified if it can be inferred from the IP address.
+                        minLength: 1
+                        type: string
+                      mode:
+                        description: Mode depicts the mode that is used for the macvlan interface. The default mode is "vepa".
+                        enum:
+                        - MacvlanModeBridge
```

Comment:
This is what the customer writes in the yaml. While `MacvlanModeBridge` is a really good name for a constant in `types.go`, we typically don't add that kind of prefix in the actual value of the string. `Bridge` would be adequate, and the same with the rest.
VEPA is an acronym so I think it should be `VEPA` instead of `Vepa`, but I don't care too much about that detail, either way is gfine by me.